gulp/tasks/copy.js

```diff
@@ -23,33 +23,38 @@ function _genericCopy( src, dest ) {
 }
 
 gulp.task( 'copy:icons', function() {
-  return _genericCopy( config.icons.src, config.icons.dest );
+  var icons = configCopy.icons;
+  return _genericCopy( icons.src, icons.dest );
 } );
 
 gulp.task( 'copy:vendorfonts', function() {
-  return _genericCopy( config.vendorfonts.src, config.vendorfonts.dest );
+  var vendorFonts = configCopy.vendorfonts;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2123/commits/f833fd8971ae3a992e54a515d6a994f8c0a771dc